libraries/Microsoft.Bot.Builder.LanguageGeneration/LGTemplate.cs

```diff
@@ -64,7 +64,18 @@ public LGTemplate(LGFileParser.TemplateDefinitionContext parseTree, string sourc
         /// </value>
         public LGFileParser.TemplateDefinitionContext ParseTree { get; }
 
-        private string ExtractBody(LGFileParser.TemplateDefinitionContext parseTree) => parseTree.templateBody()?.GetText();
+        private string ExtractBody(LGFileParser.TemplateDefinitionContext parseTree)
```

Comment:
i think we should fix this in parseTree, not here, the last one token EOF should be removed